src/unix/bsd/freebsdlike/mod.rs

```diff
@@ -1156,6 +1156,8 @@ pub const PTHREAD_MUTEX_ERRORCHECK: ::c_int = 1;
 pub const PTHREAD_MUTEX_RECURSIVE: ::c_int = 2;
 pub const PTHREAD_MUTEX_NORMAL: ::c_int = 3;
 pub const PTHREAD_MUTEX_DEFAULT: ::c_int = PTHREAD_MUTEX_ERRORCHECK;
+pub const PTHREAD_MUTEX_STALLED: ::c_int = 0;
+pub const PTHREAD_MUTEX_ROBUST: ::c_int = 1;
```

Comment:
Are those actually declared on DragonFlyBSD? I couldn't find any references.

Reply:
they are not, I moved it to FreeBSD instead.